src/main.js

```diff
@@ -6,6 +6,35 @@ import BlogLayout from '~/layouts/Blog.vue'
 import Section from '~/components/Section.vue'
 import Card from '~/components/Card'
 
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { config, library } from '@fortawesome/fontawesome-svg-core'
+import { faWindowClose } from '@fortawesome/free-regular-svg-icons'
+import {
+	faChevronRight,
+	faEye,
+	faStopwatch,
+	faSyncAlt,
+	faTrashAlt,
+	faUsers,
+	faVideo
+	} from '@fortawesome/free-solid-svg-icons'
+import '@fortawesome/fontawesome-svg-core/styles.css'
+
+config.autoAddCss = false;
+library.add(
+	faChevronRight,
+	faEye,
+	faStopwatch,
+	faSyncAlt,
+	faTrashAlt,
+	faUsers,
+	faVideo
+)
+
+export default function (Vue) {
+	Vue.component('font-awesome', FontAwesomeIcon)
+}
+
```

Comment:
You're defining here a default export, but there already is one on line 48 below, you need to remove this and add the one line to that function